dashboard/15-final/app/dashboard/(overview)/page.tsx

```diff
@@ -19,6 +20,9 @@ export default async function Page() {
 
   return (
     <main>
+      <h1 className={clsx(lusitana.className, 'mb-4 text-xl md:text-2xl')}>
```

Comment:
You can use a template literal for this. 

```suggestion
      <h1 className={`${lusitana.className} mb-4 text-xl md:text-2xl`}>
```